README.md

```diff
@@ -6,14 +6,14 @@ This project is used by the recruitment team for their Rails interview process.
 
 To get started with the app, clone the repo and then install the needed gems running the setup script:
 
-```
-$ ./bin/setup
+```bash
+./bin/setup
```

Comment:
I did remove the `$` sign because if you use the copy/past GitHub tool it will copy the `$` as well and we don't want that